stable/grafana/values.yaml

```diff
@@ -158,7 +158,7 @@ initChownData:
   ##
   image:
     repository: busybox
-    tag: 1.30
+    tag: "latest"
```

Comment:
I think busybox:1.30 should be ok.
`$ docker pull busybox:1.30
1.30: Pulling from library/busybox
697743189b6d: Pull complete
Digest: sha256:061ca9704a714ee3e8b80523ec720c64f6209ad3f97c0ff7cb9ec7d19f15149f
Status: Downloaded newer image for busybox:1.30
`

Reply:
i agree that 1.30 is ok. i'm not a fan of latest. "1.30" would probably do the trick.

Reply:
set `1.30`